src/app/components/client/dialog/Dialog.module.scss

```diff
@@ -2,48 +2,87 @@
 
 .dialog {
   background-color: $color-white;
-  position: relative;
```

Comment:
Are you using a plugin or something to reorder these rules? If so, it might be good to add that to one of our configs so that it's the same for everybody - right now it's a bit hard to see what's actually changed, and what's just been rearranged.

Reply:
Yes. I’ll bring it up in our weekly tech discussion and disable it for now.